lib/helpers.js

```diff
@@ -117,14 +117,11 @@ function crudSubcommands(crudOptions, module) {
     }
 
     if (_.includes(TAKES_OPTIONS, command)) {
```

Comment:
This looks like breaking change. defaulting to `{}` still won't cover the else block that was removed here.